src/components/NcListItem/NcListItem.vue

```diff
@@ -396,7 +396,7 @@ export default {
 		 */
 		actionsAriaLabel: {
```

Comment:
i would suggest to create computed prop `actionsAriaLabel` and create a value like `t('Actions for item with title {propertyName}', { propertyName: this.title})`
And drop `actionsAriaLabel` prop. That would be a small breaking change, i think with time we can replace all remaining places.

Reply:
We could do that without a breaking change, by keeping the `actionsAriaLabel` prop and either use the prop if provided or the computed if not. Is this fine? Then I will adjust the PR.

Reply:
A bit "duplication" but fine -> no breaking change.

Reply:
Done.